### ISSUE TITLE
zoom.get_request() error in latest version

### ISSUE THREAD

**Issue by tracy-u** [move-coop/parsons#472]
I am encountering an error for `zoom.get_meetings()` on the `latest` docker image tag. The script runs as expected with the `0.16.0` docker image tag.

```
Traceback (most recent call last):
2021-01-12 09:16:01 AM  File "vendors/zoom/zoom_meeting_to_redshift_w_meeting_id.py", line 96, in <module>
2021-01-12 09:16:01 AM    main()
2021-01-12 09:16:01 AM  File "vendors/zoom/zoom_meeting_to_redshift_w_meeting_id.py", line 41, in main
2021-01-12 09:16:01 AM    host_mtg = zoom.get_meetings(host)
2021-01-12 09:16:01 AM  File "/src/parsons/zoom/zoom.py", line 121, in get_meetings
2021-01-12 09:16:01 AM    tbl = self._get_request(f'users/{user_id}/meetings', 'meetings')
2021-01-12 09:16:01 AM  File "/src/parsons/zoom/zoom.py", line 44, in _get_request
2021-01-12 09:16:01 AM    self.refresh_header_token()
2021-01-12 09:16:01 AM  File "/src/parsons/zoom/zoom.py", line 37, in refresh_header_token
2021-01-12 09:16:01 AM    token = jwt.encode(payload, self.api_secret, algorithm='HS256').decode("utf-8")
2021-01-12 09:16:01 AMAttributeError: 'str' object has no attribute 'decode'
2021-01-12 09:16:03 AMFailed: The job container failed. Exit code 1
```

**Comment by ydamit** [comment]
Preliminary investigation suggests that the issue is that `jwt.encode()` returns a string, rather than bytes, which is what `decode()` expects.

I'm hoping to be able to propose a solution later today - but I'm not against someone else swooping in they have one (or better understand the problem)!

**Comment by ydamit** [comment]
After some investigation, I now believe that the problem is the version of of the `jwt` package must be different between the two. I can't figure out how the code, as written, ever worked, because it looks like the following are true for using this package:
- using the `encode()` method requires instantiating the `JWT()` class, which [our Zoom class](https://github.com/move-coop/parsons/blob/master/parsons/zoom/zoom.py) doesn't do
- [that method](https://github.com/GehirnInc/python-jwt/blob/264fc8f3f75d88e5f0c021a97e08ad5edd2a6007/jwt/jwt.py#L40) doesn't seem to _ever_ have taken a named argument called `algorithm`, it's seems to have been `algo` as long as it's existed
- the `signing_key` argument must be an instance of the class the package uses for encrypted data (submitting plain texts results in an error)
- the output of the `encode()` method is already a string, and therefore can't (and doesn't need to) be decoded

That last one is where we're seeing the error. But I think the core of the issue is the previous point: that the `jwt` package seems to require that the key be encrypted.

I'll do some local testing to make sure it works and then submit a PR.

**Comment by ydamit** [comment]
Ok, now I'm realizing that the real issue is that this code was written for use with the[ PyJWT package](https://github.com/jpadilla/pyjwt), which uses the same name for importing as the one I've been referencing, and which now seems to come built-in with Python: [python-jwt](https://github.com/GehirnInc/python-jwt).

I _believe_ that sorting out that ambiguity should solve this, or at least lead to progress.

**Comment by jburchard** [comment]
Addressed #478.
